### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by SSEKPIUS**
- [ ] Initialise React app.
- [ ] The building blocks of your app should be set as reusable components. Create a directory for your components.
- [ ] Add [React Router](https://reactrouter.com/web/guides/quick-start) and set two <Route>s and <Link>s for the app's navigation:
- [ ] Books - the default view:
- [ ] Should display the list of books (empty at this point but it should be ready for the data) with the Remove button (no functionality yet).
- [ ] Create a component called Book for displaying a single book (receive a title and an author as prop) and reuse it in a component that displays a list of books.
- [ ] Should display a form for adding a book (no functionality yet).
- [ ] Create a separate component for this form (with inputs for a title and an author [IMPORTANT in the design you can see an input for a category - please replace it with an author.]).
- [ ] Categories:
- [ ] Should display a button "Check status" only.
- [ ] Styling is not required at this point.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
+/* eslint-disable react/prop-types */
```
Comment (rustam-tolipov) [inline src/components/Book.js]: - [x] Students are not permitted to use ESLint-disabler. I strongly encourage you to remove this line and address the indicated error.

File: src/components/Categories.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+import React from 'react';
+
+const Categories = () => <>Categories</>;
```
Comment (rustam-tolipov) [inline src/components/Categories.js]: - [x] In the Categories component, there should be a button. When clicked, it should display "Check status". Please add this button.

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @SSEKPIUS,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
You are totally free to remove this:

NOTE: we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.

_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by danmainah — APPROVED** [review]
Hi @SSEKPIUS , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 

## Highlights
✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**